server/models/assets.js

```diff
@@ -6,6 +6,7 @@ const path = require('path')
 const fs = require('fs-extra')
 const _ = require('lodash')
 const assetHelper = require('../helpers/asset')
+const BB = require('bluebird')
```

Comment:
Use `const Promise =` for consistency across the project.

Reply:
fixed